lib/model_sorter.rb

```diff
@@ -0,0 +1,25 @@
+class ModelSorter
+
+  def self.sort(scope, params)
+    sort_attribute = params[:sort_attribute]
+    sort_direction = params[:sort_direction]
+
+    if sort_attribute.present? && sort_direction.present?
+      sort_scope(scope, sort_attribute, sort_direction)
+    else
+      scope
+    end
+  end
+
+private
+
+  def self.sort_scope(scope, attribute, direction)
+    case direction
+    when "asc", "desc"
+      scope.order("#{attribute} #{direction}")
```

Comment:
Will fix - didn't realise you could do `order(attribute => direction)`. That is new in 4 by the looks of it. (didn't work in tempo which is 3)